### PR TITLE
Fix Page Layout components scrolling issue

### PR DESCRIPTION
**Pull request by CarlosCortizasCT**
#### Summary

Fix Page Layout components scrolling issue

#### Description

This is a follow-up of #3343, where we updated some styles for page layout components to address a problem when rendering them inside a Custom View which makes them incorrectly manage scroll. Instead of the header to be fixed and the content scrollable, the whole page component is scrollable. It was spotted thanks to MC frontend e2e tests.

The right solution is to include some global styles (that we are including in the host applications) also in the Custom View content iframe.

### CODE REVIEW

File: packages/application-components/src/components/internals/page.styles.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { designTokens as appKitDesignTokens } from '../../theming';
 
 export const ContentWrapper = styled.div`
   flex: 1;
-  flex-basis: 0%;
+  flex-basis: 0;
```
Comment (CarlosCortizasCT) [inline packages/application-components/src/components/internals/page.styles.ts:6]: Revert the change introduced in the previous PR.

File: packages/application-components/src/components/main-pages/internals/main-page.styles.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ export const Divider = styled.hr`
 
 export const MainPageContent = styled.div`
   flex: 1;
-  flex-basis: 0%;
+  flex-basis: 0;
```
Comment (CarlosCortizasCT) [inline packages/application-components/src/components/main-pages/internals/main-page.styles.ts:16]: Revert the change introduced in the previous PR.

File: packages/application-shell/src/components/custom-view-shell/custom-view-shell.tsx [hunk 3]
```diff
@@ -184,58 +185,61 @@ function CustomViewShell(props: TCustomViewShellProps) {
...
-              hostUrl={hostUrl}
-              customViewConfig={hostContext.customViewConfig}
-            >
-              <CustomViewShellAuthenticated
-                dataLocale={hostContext.dataLocale}
-                environment={window.app}
-                messages={props.applicationMessages}
-                projectKey={hostContext.projectKey}
+    <>
+      <GlobalStyles />
```
Comment (CarlosCortizasCT) [inline packages/application-shell/src/components/custom-view-shell/custom-view-shell.tsx:189]: Basically we only include this `GlobalStyles` component which includes a 100% height for the main HTML document content.

Comment (emmenko) [reply packages/application-shell/src/components/custom-view-shell/custom-view-shell.tsx:189]: Ah good catch, thanks!

**Review by emmenko — APPROVED** [review]

**Review by kark — APPROVED** [review]
Thank you!
